lib/gitlab/client/gitlab_ci_yml_templates.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class Gitlab::Client
+  # Defines methods related to gitlab_ci.yml templates.
+  # @see https://docs.gitlab.com/ce/api/templates/gitlab_ci_ymls.html
+  module GitlabCiYmlTemplates
+    # Get all gitlab_ci_yml templates.
```

Comment:
```suggestion
    # Get all `gitlab-ci.yml` templates.
```

(I assume the docs support markdown, so the backticks would work?)